docs/podman-logs.1.md

```diff
@@ -15,7 +17,11 @@ any logs at the time you execute podman logs
 
 **--follow, -f**
 
-Follow log output.  Default is false
+Follow log output.  Default is false.
+
+Note: If you are following a container which is removed on exit
+`podman run --rm ...`, then there is a chance the the log file will be removed
```

Comment:
I don't think this is specific to `--rm` - I imagine you can trigger this with `rm -f` on any running container. Maybe we should just say "following a container which is removed"?